src/components/QuestDetails/index.tsx

```diff
@@ -206,9 +206,7 @@ export default function QuestDetails({
       <div className={cn(styles.container, classNames?.content)}>
         {sticker}
         <div className={cn('title', styles.title)}>{title}</div>
-        <div className={cn('body-sm', 'color-neutral-400', styles.description)}>
-          {description}
-        </div>
+        {description}
```

Comment:
is there a way we can preserve the above div styles if they only pass in a string for the description?

Reply:
Yup sorted, instead if we would need to style `MarkdownDescription`, we can just pass `classNames.root` and we can handle it. The functionality is reverted to the default container of `QuestDetails`